data/mods/Xedra_Evolved/player/professions.json

```diff
@@ -263,6 +262,190 @@
+  },
+  {
+    "type": "profession",
+    "id": "xe_dreamer",
+    "name": "Mobile Taks Force: Theta 1 \"Middle-Management\"",
+    "description": "Theta 1, also known as \"kindergarten\" between other task forces, is used when missions are expected to be too dangerous to send people or when reconnaissance can't obtain any important information about the target, which is why otherworldly creatures, summoned by dreamers, are sent.  It's not the best course of action, but the unit's nearly zero fatality rate makes it worth using.",
+    "points": 7,
+    "skills": [
+      { "level": 2, "name": "gun" },
+      { "level": 1, "name": "dodge" },
+      { "level": 2, "name": "melee" },
+      { "level": 4, "name": "deduction" }
+    ],
+    "spells": [
+      { "id": "banish_nether_monsters", "level": 7 },
+      { "id": "spell_dreamer_clairvoyance_eff", "level": 7 },
+      { "id": "summon_shifter", "level": 7 },
+      { "id": "summon_winch", "level": 7 }
+    ],
+    "traits": [ "DREAMER" ],
+    "items": {
+      "both": {
+        "ammo": 100,
+        "items": [
+          "under_armor",
+          "under_armor_shorts",
+          "tac_helmet",
+          "glasses_bal",
+          "gloves_tactical",
+          "boots_combat",
+          "holster",
+          "legrig",
+          "tacvest",
+          "socks"
+        ],
+        "entries": [
+          { "item": "xedra_jumpsuit", "variant": "xedra_jumpsuit_dreamer" },
+          { "item": "water_clean", "ammo-item": "water_clean", "container-item": "canteen" },
+          { "group": "military_ballistic_vest_light" },
+          { "item": "fn57_sup", "ammo-item": "57mm_ss", "charges": 20, "container-item": "shoulder_holster" },
+          { "item": "legpouch_large", "contents-group": "suppressed_fn57" }
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "sports_bra", "boxer_shorts" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "xe_inventor",
+    "name": "Mobile Taks Force: Nu 3 \"Techno Savvy\"",
+    "description": "Despite the \"Task Force\" in their name, Nu 3 is rarely called into the actual mission; their main task is reconnaissance and developing tools to support another task force, even if these tools last only a few weeks.  But when they do accept a task, it's almost guaranteed to be successful.  It should've been your first day here, but something went really wrong.",
+    "points": 7,
+    "skills": [
+      { "level": 1, "name": "gun" },
+      { "level": 1, "name": "melee" },
+      { "level": 2, "name": "fabrication" },
+      { "level": 2, "name": "mechanics" },
+      { "level": 3, "name": "electronics" },
+      { "level": 4, "name": "deduction" }
+    ],
+    "spells": [ { "id": "firmament_driver", "level": 7 }, { "id": "cable_whip", "level": 7 } ],
+    "traits": [ "INVENTOR" ],
+    "proficiencies": [ "prof_metalworking", "prof_welding_basic", "prof_elec_soldering" ],
+    "items": {
+      "both": {
+        "ammo": 100,
+        "items": [
```

Comment:
You can create a recipe book item that only spawns with this profession and has whichever Inventor recipes you want the class to start with.

Reply:
But it won't be counted into inventor/dreamsmith recipe knowledge, and when they try to research more, they effectively obtain it twice (first in the book, and secondnby actual research)